change python to python3 as alpine does not provide python
change nodejs to node to match alpine name
swap nodejs.org  for mirror returned in test so it passes
--- a/nodeenv.py
+++ b/nodeenv.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 """
@@ -513,7 +513,11 @@
         'AMD64':  'x64',  # Windows Server 2012 R2 (x64)
         'armv6l': 'armv6l',     # arm
         'armv7l': 'armv7l',
+        'armv8l': 'armv7l',
         'aarch64': 'arm64',
+        'ppc64le': 'ppc64le', # Power PC
+        's390x': 's390x', # IBM S390x
+
     }
     sysinfo = {
         'system': platform.system().lower(),
--- a/tests/nodeenv_test.py
+++ b/tests/nodeenv_test.py
@@ -24,7 +24,7 @@
     ])
     assert os.path.exists(nenv_path)
     subprocess.check_call([
-        'sh', '-c', '. {0}/bin/activate && nodejs --version'.format(nenv_path),
+        'sh', '-c', '. {0}/bin/activate && node --version'.format(nenv_path),
     ])
 
 
@@ -87,8 +87,8 @@
              'https://npm.taobao.org/mirrors/node/index.json'),
             ('npm.some-mirror.com',
              'https://npm.some-mirror.com/download/release/index.json'),
-            ('',
-             'https://nodejs.org/download/release/index.json')]
+            ('https://unofficial-builds.nodejs.org/download/release',
+             'https://unofficial-builds.nodejs.org/download/release/index.json')]
     with open(os.path.join(HERE, 'nodejs_index.json'), 'rb') as f:
         def rewind(_):
             f.seek(0)
